src/adapters/search_suggestions_adapter.js

```diff
@@ -1,26 +1,49 @@
-import webChannel from '../lib/web_channel';
+import State from 'ampersand-state';
 import _c from 'lodash/collection';
 import SearchSuggestions from '../collections/search_suggestions';
+import webChannel from '../lib/web_channel';
+
+const SearchSuggestionsAdapter = State.extend({
+  props: {
+    engine: 'string',
+    searchTerm: 'string'
+  },
+
+  initialize () {
+    this.maxLocalSuggestions = 2;
+    this.maxCombinedSuggestions = 5;
```

Comment:
Someday when we have such a thing that seems like a good idea :smile:. For now at least they aren't magic numbers!